ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/cv-pathfinder-edge3B.yml

```diff
@@ -158,14 +158,14 @@ ethernet_interfaces:
     enabled: false
   description: Colt_10423
 - name: Ethernet52
-  switchport:
-    enabled: false
+  type: routed
```

Comment:
This is wrong, please make sure to rebase and not overwrite whatever creates this.

Reply:
it was rebased already but indeed I created another helper function where this was carried over